Dockerfile

```diff
@@ -7,30 +7,71 @@ ENV COMPOSER_PROCESS_TIMEOUT 3600
 
 WORKDIR /code/
 
-COPY docker/php-prod.ini /usr/local/etc/php/php.ini
```

Comment:
Pro jistotu bych zabloknul PHP verzi na aktuální (7.4) aby se to nerozbilo až bude 7.5... řešíme to docela často...

Reply:
ok.

Reply:
@tomasfejfar point OK, ale 7.5 sa nedočkají ani naše děti :D

Reply:
Ajo :)